process/api.py

```diff
@@ -87,6 +99,161 @@ class CollectionViewSet(viewsets.ViewSetMixin, ListAPIView):
         "completed_at",
     ]
 
+    @csrf_exempt
+    @action(methods=["post"], detail=False)
+    def new(self, request):
```

Comment:
Use `def create`, which has special meaning and doesn't require `@action` (at least, when used with the correct superclass).

Reply:
Yeap, I only changed this one because it was complaining about `create` existing already